packages/kolibri-tools/lib/webpack_json.py

```diff
@@ -15,6 +15,9 @@
 
 logger = logging.getLogger("webpack_json")
 logger.setLevel(level=logging.INFO)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
```

Comment:
what does 'ch' stand for?

Reply:
It stands for: "Richard copied this from the Python docs" - https://docs.python.org/3/howto/logging-cookbook.html#using-logging-in-multiple-modules

Reply:
super minor, recommend updating the variable name before merging :P

Reply:
To be honest, I'm not even sure this fixes the logging problem.